openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx

```diff
@@ -84,14 +87,13 @@ const ProfilerDashboard: React.FC<ProfilerDashboardProps> = ({
 }) => {
   const { getEntityPermission } = usePermissionProvider();
   const history = useHistory();
-  const { entityTypeFQN, dashboardType } = useParams<Record<string, string>>();
+  const { entityTypeFQN, dashboardType, tab } =
+    useParams<Record<string, string>>();
```

Comment:
Can we use strict type here?
```suggestion
    useParams<{entityTypeFQN:string, dashboardType:string, tab:string}>();
```